src/writer/settings.ts

```diff
@@ -80,6 +81,26 @@ export async function verify(body): Promise<any> {
     return Promise.reject(`max number of strategies is ${strategiesLimit}`);
   }
 
+  try {
+    const strategiesList = await (await fetchWithKeepAlive(`${scoreAPIUrl}/api/strategies`)).json();
```

Comment:
Maybe we should call it every 5 mins instead of calling it everytime, but can make things look complicated 🙈